subprojects/docs/src/docs/release/notes.md

```diff
@@ -32,6 +32,23 @@ details of 2
 
 ## n
 -->
+## Security 
+
+### Removed debug logging for environment variables
+
+Gradle allows you to start processes in various ways, for example via `Project.exec`, `Project.javaexec`, running tests or starting a Gradle daemon.
+When you enabled debug logging for the Gradle process, then it used to log all environment variables of the process about to start.
+Often there are secret values or credentials in the environments, especially when running builds as part of continuous integration.
+Those secrets leaked to the log output when running a build with debug logging enabled.
+
+Now Gradle doesn't log the environment anymore on debug when starting processes, which decreases the probability to leak secrets.
+
+Note that many CI servers, like Jenkins and Teamcity, mask secrets in the captured logs.
```

Comment:
How about

"Note that many CI servers, like Jenkins and Teamcity, mask secrets in the captured logs. Still, we recommend limiting the usage of debug level logging to debugging on a local machine."

Reply:
👍